packages/modelviewer.dev/src/docs-and-examples/create-html.ts

```diff
@@ -209,19 +243,18 @@ function createSubcategory(
   }
 }
 
-export function convertJSONToHTML(
-    json: any[], docsOrExamples: 'docs'|'examples') {
```

Comment:
👍

Reply:
what is the thumbs up for?

Reply:
I like that you managed to remove the `docsOrExamples` argument.

Reply:
cool cool